### DISCUSSION TITLE
Feature Request: Community-submitted game manifests

### DISCUSSION THREAD

**Discussion by redactedscribe** [MikeMaximus/gbm#279] [Ideas]
It would be cool if there was a way to submit manifests to a community pool of browsable manifests in cases where the Official list and Ludusavi don't contain a given game, or for some reason the manifest isn't adequate, or maybe there the manifest would be specifically for backing up the game's screenshots and its screenshots alone, for example. Perhaps it would be also possible to rate these user-submitted manifests so others could get an idea of which one of the potentially several that exist for a game is the better one.

Maybe there'd need to be extra warnings or hoops to jump through, so that users actually read the manifest's settings, as some could be low-quality submissions and potentially just wrong.

Once I have more time, I may add to this suggested feature.

Thanks.

GBM 1.3.1

**Comment by MikeMaximus** [comment]
Thanks for the suggestion.

I'm going to move the issue to the discussion section as this is something that would be more of a long term feature addition.
